packages/client/src/components/ratings/RatingBreakdownItems/RatingBreakdownItemsAlt.tsx

```diff
@@ -0,0 +1,41 @@
+import RectangularRatingIndicator from '@components/ratings/RectangularRatingIndicator'
+import { Rating } from '@futureproof/typings'
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+
+interface Props {
+  susRatings : number[]
```

Comment:
Could this data be combined into an array of objects so that we know every value has a name? At the moment the lengths aren't guaranteed to be equal.

Reply:
Ok.